app/parse.py

```diff
@@ -1,9 +1,42 @@
+import time
 from dataclasses import dataclass
 from urllib.parse import urljoin
 
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from selenium.common import (
+    NoSuchElementException,
+    TimeoutException,
+    ElementNotInteractableException,
+)
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as exc
+from selenium.webdriver.support.wait import WebDriverWait
+
+from app import writer
+from app.generator import CSVGenerator
 
 BASE_URL = "https://webscraper.io/"
 HOME_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/")
+COMPUTERS_URL = urljoin(HOME_URL, "computers")
+LAPTOPS_URL = COMPUTERS_URL + "/laptops"
+TABLETS_URL = COMPUTERS_URL + "/tablets"
+PHONES_URL = urljoin(HOME_URL, "phones")
+TOUCH_URL = PHONES_URL + "/touch"
+
+URLS = [
+    HOME_URL,
+    COMPUTERS_URL,
+    LAPTOPS_URL,
+    TABLETS_URL,
+    PHONES_URL,
+    TOUCH_URL,
+]
+
+
+def init_driver() -> webdriver.Chrome:
+    driver = webdriver.Chrome()
```

Comment:
Consider adding options to configure the WebDriver, such as setting a headless mode for faster execution and reduced resource usage, especially if the script is intended to run in a server environment.